### ISSUE TITLE
Can I use it with an arduino?

### ISSUE THREAD

**Issue by Allmight0000** [eclipt69/neural-network-aimbot#2]
Can I use it with an arduino?

**Comment by eclipt69** [comment]
Yes, just take the screen's width and height values and the yolo's x and y values, than it's just passing them to your arduino program.
